tsconfig.json

```diff
@@ -12,7 +12,8 @@
     "importHelpers": true,
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "useUnknownInCatchVariables": false,
```

Comment:
Since the code contains `catch` and the error type doesn't correctly validate, I was forced to disable this check.